util/exercise/src/cmd/update.rs

```diff
@@ -85,7 +86,7 @@ fn apply_diffs(exercise_name: &str, diffs: &HashSet<String>, tests_content: &str
         "{}\n{}",
         tests_content,
         diffs
-            .iter()
+            .par_iter()
```

Comment:
This scales with `diffs`, which is bounded to be less than or equal to the number of tests for an exercise. Most exercises have few tests:

```sh
$ ls exercises/*/tests/*.rs | xargs rg -Fc '[test]' | awk 'BEGIN {FS=":"; OFS=","} {print $2, $1}' | sort -rn | head -n 5
Sun Feb 24 15:18:07 CET 2019
50,exercises/clock/tests/clock.rs
43,exercises/decimal/tests/decimal.rs
37,exercises/custom-set/tests/custom-set.rs
29,exercises/forth/tests/forth.rs
28,exercises/poker/tests/poker.rs
```

Therefore, for the same reasons as above, it's better to leave this single-threaded.